tests/array_interoperability_test.py

```diff
@@ -100,6 +100,7 @@ def testJaxRoundTrip(self, shape, dtype, take_ownership, gpu):
      for shape in all_shapes
      for dtype in dlpack_dtypes))
   @unittest.skipIf(not tf, "Test requires TensorFlow")
+  @jtu.skip_on_devices("rocm") # relevant dlpack protocol is N/A for ROCm ATM
```

Comment:
How is this related to the callback change?

Reply:
It is not (sorry).  But it is necessary for ROCm.  I didn't really want to create a separate PR for this.

Reply:
Okay, in that case could you add a `TODO(sharadmv,phawkins):...` and reference a GH issue?

Reply:
Just to clarify, should I create a GH issue to add rocm support for dlpack and reference that issue directly in the code?

Reply:
Done